### PR TITLE
Improve site styling

### PR DESCRIPTION
**Pull request by LissaGreense**
I did some clean-up in css, introduced variables and unify coloring.

### CODE REVIEW

File: frontend/src/components/movieMenu.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ import {
 } from "../constants/paths.ts";
 import React, { useEffect, useState } from "react";
 import { getAvatar } from "../connections/internal/user.ts";
+import "./movieMenu.css";
+import { Button } from "primereact/button";
```
Comment (adlbrtbsr) [inline frontend/src/components/movieMenu.tsx:20]: remove unused import

File: frontend/src/components/movieMenu.css
Comment (adlbrtbsr) [inline frontend/src/components/movieMenu.css:1]: do we need an empty file in repo?

File: frontend/src/pages/registerPage.css
Comment (adlbrtbsr) [inline frontend/src/pages/registerPage.css:1]: another empty file, still not sure if we need them

File: frontend/src/components/movieMenu.css
Comment (LissaGreense) [reply frontend/src/components/movieMenu.css:1]: Removed ;)

File: frontend/src/pages/registerPage.css
Comment (LissaGreense) [reply frontend/src/pages/registerPage.css:1]: As u wish

**Review by adlbrtbsr — APPROVED** [review]
Good job!!! :D

File: frontend/src/components/movieMenu.css
Comment (adlbrtbsr) [reply frontend/src/components/movieMenu.css:1]: thanks

File: frontend/src/pages/registerPage.css
Comment (adlbrtbsr) [reply frontend/src/pages/registerPage.css:1]: thanks

**Pull request merged**